docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -0,0 +1,108 @@
+# V2 Catalog Products and V3 Catalog APIs
+
+<div class="otp" id="no-index">
+
+### On this page
+- [V3 improvements](#v3-improvements)
+- [Difference between V2 Catalog Products and V3 Catalog APIs](#difference-between-v2-catalog-products-and-v3-catalog-apis)
+- [Related resources](#related-resources)
+
+</div>
+
+V2 Catalog Products and V3 Catalog REST APIs allow you to manage your store's physical and digital products.
+
+Both the V2 Catalog Products and V3 Catalog APIs authenticate with OAuth and are designed to be used concurrently within a single application; however, data representation is significantly different.
+
+V2 and V3 REST APIs are not fully compatible. For differences in resources, see [Difference between V2 Catalog Products and V3 Catalog APIs](#difference-between-v2-catalog-products-and-v3-catalog-apis). When resources are available through both APIs, we recommend using the V3 API as it contains performance optimizations and usability improvements.
+
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+>
+> * BigCommerce will not introduce new resources in V2 APIs.
+> * BigCommerce will eventually migrate all existing V2 resources to V3.
+> * Not all V2 resources have V3 counterparts. 
+
+</div>
+</div>
+</div>
+
+## V3 improvements
+
+* You can perform most tasks with fewer API calls; for example, you can create a product with variants and custom fields in a single request.
+* You can include subresources within a request.
+* Each V3 resource includes a meta object, simplifying pagination.
+* V3 Brands, Categories, Products, and Product Variants expose a metafields resource for use by developers to store custom data.
+* V3 API is optimized for performance. In general, data can be sent, received, and processed faster via V3, relative to V2.
+
+## Difference between V2 Catalog Products and V3 Catalog APIs
+
+### Product variants and modifiers
+
+V3 Catalog introduced a new option model which includes two types of options: [variant options](https://developer.bigcommerce.com/api-docs/store-management/products-overview#variant-options) and [modifier options](https://developer.bigcommerce.com/api-docs/store-management/products-overview#modifier-options). The new option model  simplifies the creation and management of variant prices and modifier adjusters and removes the need to use complex rules, in all but some cases.
+
+|Option type |Example|
+|-|-|
+|**Variant options** are any choices that the shopper needs to make that will result in selecting a variant.|Item’s size and color.|
+|**Modifier options** are any choices that the shopper can make to change how the merchant fulfills the product. |Shipping insurance and special engraving.|
+
+In V3 Catalog, variants and modifiers are attached directly to the product, without the need to create an option set beforehand. 
```

Comment:
I think it should be rephrased, because product options/modifiers created via v 3 API belong to single product. and can not be shared between/attached to other products. If you need the same option (ex. Color) assign to other products, you have to created it under these products as their own.
**Note:** in catalog v3 model we have shared options/modifiers which can be shared between products, but for now they are available only via CP. Public APIs for them would be available in the end of 2021.